paper-autocomplete-chips.js

```diff
@@ -365,27 +365,37 @@ class PaperAutocompleteChips extends translatable(PolymerElement) {
 	 */
 	_clearItemSelection(event) {
 		const item = event.model.item,
```

Comment:
```suggestion
		const chip = event.model.item,
```

Even better might be to do `<template is="dom-repeat" items="[[ enrichChips(selectedItems.*, textProperty) ]]" as="chip">` and call it chip everywhere. This also hints that "enrichChips" should be called something like "createChips" instead.